packaging/irods_prc_tests/test_rule_engine_plugin_metadata_guard_atomic.py

```diff
@@ -124,5 +124,22 @@ def test_guard_atomic_operations_editor_list__issue_38(self):
             obj.metadata.apply_atomic_operations( AVUOperation(operation='add', avu=iRODSMeta('irods::editorlist', 'editorlist')))
             self.assertEqual(obj.metadata['irods::editorlist'], iRODSMeta('irods::editorlist', 'editorlist'))
 
+        # remove test user from editor list
```

Comment:
comment is... not quite right?

Reply:
Yep, that's an oopsie on my part; copy-pasted a little too fervently...

Reply:
Oh wait, 127? Where'd I goof here? Unless coll.metadata[] doesn't do what I think it does?

Reply:
i think maybe it should just say 'alice' user instead of test user - and that would be sufficient

Reply:
Ah ok, I was trying to be consistent with 110, but that's clearer. Let me do that too...